private/aws-restjson-validation-server/package.json

```diff
@@ -11,7 +11,7 @@
     "build:types:downlevel": "downlevel-dts dist-types dist-types/ts3.4",
     "clean": "rimraf ./dist-* && rimraf *.tsbuildinfo",
     "prepack": "yarn run clean && yarn run build",
-    "test": "jest"
+    "test": "vitest run"
```

Comment:
`Makefile` and `vitest.config.protocols.integ.ts` needs updating when this happens to ensure these tests run.

I don't think these pass, because there is test code that references jest.

Reply:
this needs a codegen update upstream to remove the global `jest` and import `vi`.

Reply:
I would just leave it out of this PR.

Reply:
Reverted in [`b6b04de` (#6627)](https://github.com/aws/aws-sdk-js-v3/pull/6627/commits/b6b04decb08f74c9d847a36a9b6622f3956717e1)